state/src/interface/order_book.rs

```diff
@@ -371,50 +357,22 @@ impl StateInner {
 // -----------
 
 impl StateInner {
-    /// Checks whether or not a locally managed order is matchable
-    fn is_order_matchable<T: TransactionKind>(
```

Comment:
now that we're caching the bulk of this check, we only need to check whether the task queue is paused.